server/web/forms.py

```diff
@@ -18,3 +19,22 @@ def validate_password(self, field):
         user = User.query.filter_by(email=self.email.data).first()
         if not user or not user.check_password(self.password.data):
             raise ValidationError('Invalid Email/Password')
+
+
+class SignUpForm(FlaskForm):
+    company_name = StringField('Company Name', validators=[DataRequired()])
```

Comment:
Forgot to have `Length` validation on the requirements. Set the max to whatever the db model is for `company_name, first_name, last_name, email`